### ISSUE TITLE
Can't use on server

### ISSUE THREAD

**Issue by Xuleos** [osyrisrblx/rbxts-projectile#3]
Because of the forced use of RunService.RenderStepped, it isn't possible to use @rbxts/projectile at all on the server.

(my use case was to fire a projectile locally on the client, and then ask the server to fire an identical one as well with a custom renderer so that I can handle onTouched on the server)

Maybe add an option in the config to allow us to choose what RunService event should be used for the projectile? With the default being RenderSteppeod of course to avoid making a breaking change

**Comment by osyrisrblx** [comment]
Interesting use case.

Generally, what I've done in the past is use a RemoteEvent `client -> server -> all other clients` to communicate the creation of a projectile with position, velocity, acceleration, etc.

Is this for a security type thing?

**Comment by Xuleos** [comment]
Yes, I'd much rather be able to handle hits on the server for security

**Comment by osyrisrblx** [comment]
Hmm maybe the package should not make the `RenderStepped` connection for you at all. Could just supply a static `update` function and have users create their own connections.

**Comment by Xuleos** [comment]
seems good
